modules/users/server/routes/admin.server.routes.js

```diff
@@ -4,22 +4,28 @@
  * Module dependencies
  */
 var adminPolicy = require('../policies/admin.server.policy'),
-  admin = require('../controllers/admin.server.controller');
+  admin = require('../controllers/admin.server.controller'),
+  passport = require('passport'),
+  express = require('express');
 
 module.exports = function (app) {
-  // User route registration first. Ref: #713
-  require('./users.server.routes.js')(app);
+  var router = express.Router();
 
-  // Users collection routes
-  app.route('/api/users')
+  //Set JWT Auth for all Admin Routes
+  router.all('*', passport.authenticate('jwt', { session: false }));
+
+  router.route('/')
     .get(adminPolicy.isAllowed, admin.list);
 
   // Single user routes
-  app.route('/api/users/:userId')
+  router.route('/:userId')
     .get(adminPolicy.isAllowed, admin.read)
     .put(adminPolicy.isAllowed, admin.update)
     .delete(adminPolicy.isAllowed, admin.delete);
 
   // Finish by binding the user middleware
-  app.param('userId', admin.userByID);
+  router.param('userId', admin.userByID);
+
+  app.use('/api/admin/users', router);
```

Comment:
This is a good discussion on why not to use `/api/admin/users` https://github.com/meanjs/mean/pull/713

After a few long discussions, @rhutchison made me see the light & I feel like we shouldn't separate the User routes between Admin & normal User access. 

If the `/admin/` is removed from the API route & then removed from the client-side service, then Admin feature breaks. Perhaps, that's a hint on the reason for having to add the Express Router implementation here.

I think it adds an extra layer of complexity that doesn't add any value; and it actually forces us to create an API route (*/api/admin/users* that goes against convention.

Can we find a way to avoid these changes? I'm not opposed to a refactoring of the routes, but maybe not in this PR, or finding a way here to keep the same API for both Admins & Users.